src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-            std::vector<char *> predict_agrs_vector;
-            std::transform(predictargs.begin(), predictargs.end(), std::back_inserter(predict_agrs_vector), converter);
-
-            std::string path = "cd " + Utils::getJasmineGraphProperty("org.jasminegraph.graphsage") + " && ";
-            std::string command = path + "python3.8 predict.py ";
-
-            int argc = predictargs.size();
-            for (int i = 0; i < argc; ++i) {
-                command += predictargs[i];
-                command += " ";
-            }
+        bzero(data, 301);
+        read(sockfd, data, 300);
+        string response = (data);
+        response = Utils::trim_copy(response, " \f\n\r\t\v");
 
-            instance_logger.error("Temporarily disabled the execution of prediction.");
-            // TODO(miyurud): Temporarily commenting the execution of the following line
-            // due to missing predict.py file. Removal of graphsage folder resulted in this situation.
-            // Need to find a different way of executing Predict
-            // system(command.c_str());
-            loop = true;
-        } else if (line.compare(JasmineGraphInstanceProtocol::INITIATE_MODEL_COLLECTION) == 0) {
-            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::INITIATE_MODEL_COLLECTION, "info");
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string serverHostName = (data);
-            serverHostName = Utils::trim_copy(serverHostName, " \f\n\r\t\v");
-            instance_logger.log("Received HostName: " + serverHostName, "info");
+        if (response.compare(JasmineGraphInstanceProtocol::OK) == 0) {
+            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::OK, "info");
+            result_wr = write(sockfd, graphID.c_str(), graphID.size());
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string serverHostPort = (data);
-            serverHostPort = Utils::trim_copy(serverHostPort, " \f\n\r\t\v");
-            instance_logger.log("Received Port: " + serverHostPort, "info");
+            if (result_wr < 0) {
+                instance_logger.log("Error writing to socket", "error");
+            }
+            instance_logger.log("Sent : Graph ID " + graphID, "info");
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string serverHostDataPort = (data);
-            serverHostDataPort = Utils::trim_copy(serverHostDataPort, " \f\n\r\t\v");
-            instance_logger.log("Received Data Port: " + serverHostDataPort, "info");
+            bzero(data, 301);
```

Comment:
```suggestion
            bzero(data, INSTANCE_DATA_LENGTH + 1);
```

Reply:
`bzero()` is no longer used in the next PR to clear the `data` array. It's handled in the wrapper function without `bzero()`

Reply:
Ok